ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -970,9 +970,54 @@ open class TableViewCell: UITableViewCell {
     ///   - customAccessoryView: The view acting as an accessory view that appears on the trailing edge, next to the accessory type if provided
     ///   - accessoryType: The type of accessory that appears on the trailing edge: a disclosure indicator or a details button with an ellipsis icon
     @objc open func setup(title: String, subtitle: String = "", footer: String = "", customView: UIView? = nil, customAccessoryView: UIView? = nil, accessoryType: TableViewCellAccessoryType = .none) {
-        titleLabel.text = title
-        subtitleLabel.text = subtitle
-        footerLabel.text = footer
+        setup(title: title,
+              attributedTitle: nil,
+              subtitle: subtitle,
+              attributedSubtitle: nil,
+              footer: footer,
+              attributedFooter: nil,
+              customView: customView,
+              customAccessoryView: customAccessoryView,
+              accessoryType: accessoryType)
+    }
+
+    /// Sets up the cell with text, a custom view, a custom accessory view, and an accessory type
+    ///
+    /// - Parameters:
+    ///   - title: Text that appears as the first line of text
+    ///   - attributedTitle: Optional attributed text for the first line of text. If this is not set, the title will be used
+    ///   - subtitle: Text that appears as the second line of text
+    ///   - attributedSubtitle: Optional attributed text for the second line of text. If this is not set, the subtitle will be used
+    ///   - footer: Text that appears as the third line of text
+    ///   - attributedFooter: Optional attributed text for the third line of text. If this is not set, the footer will be used
+    ///   - customView: The custom view that appears near the leading edge next to the text
+    ///   - customAccessoryView: The view acting as an accessory view that appears on the trailing edge, next to the accessory type if provided
+    ///   - accessoryType: The type of accessory that appears on the trailing edge: a disclosure indicator or a details button with an ellipsis icon
+    @objc open func setup(title: String = "",
+                          attributedTitle: NSAttributedString? = nil,
+                          subtitle: String = "",
+                          attributedSubtitle: NSAttributedString? = nil,
+                          footer: String = "",
+                          attributedFooter: NSAttributedString? = nil,
+                          customView: UIView? = nil,
+                          customAccessoryView: UIView? = nil,
+                          accessoryType: TableViewCellAccessoryType = .none) {
+        if let attributedTitle = attributedTitle {
```

Comment:
could this block of code be one line something like  attributedTitle?:title

Reply:
titleLabel.attributedText = attributedTitle != nil ? attributedTitle : NSAttributedString(string: title)

Does that look okay? That would make any text into an attributed string though

Reply:
ahhh you are right. actually we should jsut use https://developer.apple.com/documentation/uikit/uilistcontentconfiguration ?

Reply:
This looks like a good idea. I would have to override the `defaultContentConfiguration` and add our own properties though, right? Which would include updating the demo? Should I do that in this PR?

Reply:
so it would be something like self.defaultContentConfiguration().text = "foo" in this function?

Reply:
oh wait... sorry... do our cell actually use the OS tableviewcell's label? :)

Reply:
Well there's 3 different text labels so I think I'd have to override the default config so that it knows the difference between the 3 labels?

Reply:
i don't think overiding defaultcontentconfiguration would work. looking at the tableviewcell code we created our OWN labels. so your current approach is correct.

Reply:
Sorry your previous message about using the OS labels didn't load early so I didn't see that before my reply :P Okay yes sounds good, but this discussion was productive still!